lupa/_lupa.pyx

```diff
@@ -633,12 +654,15 @@ cdef class LuaRuntime:
         # create 'python' lib
         luaL_openlib(L, "python", py_lib, 0)       # lib
         lua.lua_pushlightuserdata(L, <void*>self)  # lib udata
-        lua.lua_pushcclosure(L, py_args, 1)        # lib function
+        lua.lua_pushcclosured(L, py_args, 1)        # lib function
```

Comment:
Rather than renaming all of these in the source code, you can use explicit cnames in Cython to make it generate a different name in C than in the Cython code:
```cython
cdef extern from "whatever.h":
    """
    #if LUAU
    #define __lupa_pushcclosure(L, fn, n)  lua_pushcclosured(L, fn, n)
    #else
    #define __lupa_pushcclosure(L, fn, n)  lua_pushcclosure(L, fn, n)
    #endif
    """
    void  lua_pushcclosure "__lupa_pushcclosure" (lua_State *L, lua_CFunction fn, int n)
```